### PR TITLE
15 export a list of supported chains

### LINKED ISSUE #15: Export a list of supported chains
The users may want to check what chains are supported programmatically

### PR DESCRIPTION
**Pull request by hiletmis**
Closes #15

### CODE REVIEW

**Comment by bbenligiray** [comment]
This is painful to review. What if you had the chain aliases in `chains.json`, but converted them into chain IDs using `@api3/chains` before exporting?

**Comment by hiletmis** [comment]
@bbenligiray chain ids replaced with chain aliases

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+
+    if (chain.testnet) {
+      testnetChainIds.push(chain.id);
+    } else {
+      mainnetChainIds.push(chain.id);
+    }
+    return {
+      mainnet: mainnetChainIds.sort(),
+      testnet: testnetChainIds.sort(),
+    };
```
Comment (bbenligiray) [inline src/index.js:27]: Just return the chain IDs, don't group them as mainnets/testnets. The user can refer to `@api3/chains` for that.

Comment (hiletmis) [reply src/index.js:27]: changed the code to return the following response but cannot push it as this repo has been moved to `@api3/dao`;
`[
  '1',        '10',    '30',
  '31',       '56',    '97',
  '100',      '137',   '250',
  '420',      '1101',  '1284',
  '1285',     '1287',  '1442',
  '2221',     '2222',  '4002',
  '5000',     '5001',  '8453',
  '10200',    '42161', '43113',
  '43114',    '59140', '59144',
  '80001',    '84531', '421613',
  '11155111'
]`

**Review by bbenligiray — APPROVED** [review]
See my commits

**Comment by hiletmis** [comment]
Thanks for the improvements

**Pull request merged**